### DISCUSSION TITLE
Help Needed: Recovering a Deleted GitHub Account & Repositories

### DISCUSSION THREAD

**Discussion by deepumandal** [community/community#135151] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Hi everyone,

I’m reaching out to seek advice on a critical issue. A friend of mine recently and accidentally deleted their GitHub account, including all associated repositories, without taking any backups. This happened just a few minutes ago, and we're in urgent need of assistance to recover the account and repositories if possible.

Details:
Username: rubypoddar
Account Deletion Time: 08-08-2024

Steps We’ve Already Taken:
Checked for Local Copies: We are searching for any local copies of the repositories on collaborators' machines.
Searched for Forks: Looking into whether any forks of the repositories exist.
Help Requested:
Has anyone experienced a similar situation and successfully recovered their account or repositories?
Are there any other immediate steps we should take to increase our chances of recovery?
Any advice or guidance on how to approach GitHub Support effectively in such cases?
Are there any tools or services that might help in retrieving the deleted repositories?
We understand that the chances of recovery might be slim, but we’re hopeful that the community’s collective knowledge and experience could help us navigate this situation.

Thanks in advance for your support and guidance!

Best regards,
Deepak Mandal

**Comment by ebndev** [comment]
Hi @deepumandal,

This community is managed by GitHub employees, but it is not a support forum. The only place to go for answers to account-specific questions like this one is opening a ticket on our [Support page](https://support.github.com/contact).

This is a public forum, and the community is unable to assist with nor escalate your ticket, but we can assure you it is in the right place. It will be answered in the order it was received based on our team's resources and the amount of tickets ahead of yours.

As no other Community Discussions team members will be able to provide additional help with account related questions I am going to close this discussion. We appreciate your understanding!

**Reply by deepumandal** [reply]
@ebndev very very thanks for guidence.
